stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1588,6 +1591,8 @@ internal constructor(
         channelId: String,
         message: Message,
         isRetrying: Boolean = false,
+        skipPush: Boolean = false,
+        skipEnrichUrl: Boolean = false,
```

Comment:
We should add those parameters to the `ChannelClient::sendMessage` as well

Reply:
Updated